engine/engines.go

```diff
@@ -52,11 +52,14 @@ func Scrape(engine Engine) ([]Movie, error) {
 		// Cache responses to prevent multiple download of pages
 		// even if the collector is restarted
 		colly.CacheDir("./gophie_cache"),
-		colly.Async(true),
+		// colly.Async(true),
 		//    colly.Debugger(&debug.LogDebugger{}),
 	)
 	// Another collector for download Links
-	downloadLinkCollector := c.Clone()
+	downloadLinkCollector := colly.NewCollector(
```

Comment:
@deven96, any special reasons for splitting the collector?

Reply:
Simply testing a theory

Reply:
Okay